phys2bids/tests/test_utils.py

```diff
@@ -0,0 +1,98 @@
+"""
+    Runs unit tests on utils.py
+"""
+
+import os
+from phys2bids import utils
+
+
+# Tests check_input_dir
+def test_check_input_dir():
+    test_path = '/home/mrmeeseeks/blue/'
+    path_corrected = utils.check_input_dir(test_path)
+    assert path_corrected == test_path[:-1]
+
+
+# Tests check_input_ext
+def test_check_input_ext():
+    test_filename = 'help'
+    ext = 'py'
+    out_filename = utils.check_input_ext(test_filename, ext)
+    assert str(out_filename) == f'{test_filename}.{ext}'
+
+
+# Tests check_input_type
+def test_check_input_type():
+    test_filename = 'Test_belt_pulse_samefreq.acq'
+    test_path = '/home/travis/build/physiopy/phys2bids/phys2bids/tests/data'
```

Comment:
We should probably be using `pkg_resource.resource_filename` to get this path so that we're not hard-coding this TravisCI path in our tests! Something like this should work and be a bit more robust to different build environments:

```python
from pkg_resources import resource_filename
test_path = resource_filename('phys2bids', 'tests/data')
```

Reply:
Thank you! I didn't know about this package.